### ISSUE TITLE
No warning for out memory when opening big file

### ISSUE THREAD

**Issue by lmasini** [udo-munk/s#5]
When loading big files (in CP/M term), the s editor can run out of memory without giving any hint to the user.
It can happen in do_read() (file commands.c).

In my copy I have added this

```c    /* copy the file to the buffer */
    for (i = line; fgets(text, MAXTEXT, fp) != NULL; ++i) {
        text[strlen(text)-1] = '\0';    /* trim off the newline */
        if (b_insert(i, text) == 0) {
            s_errmsg("Out of memory on line %d", i);    <==== JUST THIS LINE TO WARN THE USER
            break;
        }
    }
```
Maybe it would be better to also mark the file as dirty and do not let the user to overwrite it.

**Comment by udo-munk** [comment]
Agreed, at least it should warn the user about out of memory.

**Comment by udo-munk** [comment]
Fixed with 8a17600
